main.cpp

```diff
@@ -52,84 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: plane
+//  action 1: the plane deploys wheels
+plane.deployWheels();
+//  action 2: the plane rotates vertically
+plane.verticalRotation();
+//  action 3: the plane consumes fuel
+plane.consumeFuel();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: Boat
+//  action 1: the boat floats over water
+boat.float();
+//  action 2: the boat rotates horizontally
+boat.horizontalRotation();
+//  action 3: the boat consumes fuel
+boat.consumeFuel(); 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: Lion
+//  action 1: the lion sleeps
+lion.sleep();
+//  action 2: the lion walks
+lion.walk();
+//  action 3: the lion hunts
+lion.hunt();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.